README.md

```diff
@@ -195,3 +195,9 @@ similar to the following, and then run `go mod tidy`.
 ```
 github.com/apache/pulsar-client-go/oauth2 v0.0.0-20220630195735-e95cf0633348 // indirect
 ```
+### Schema compatibility with JAVA client
+Due to the differences in programming language features and data representation, 
+schema incompatibility may arise between different languages.
+We have conducted an analysis of the schema compatibility between Java and Go, 
+and we have provided some feasible solutions.
```

Comment:
```suggestion
The analysis of the schema compatibility between Java and Go has been conducted and the corresponding solutions have been listed below.
```
In technical writing:
- Avoid the first-person pronouns I and we
- Use the second-person pronoun (you, your, yours, or yourself) as much as possible. The subject of an imperative sentence is understood to be you.

Details see https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.gfwms5zehcak